src/histogram.ts

```diff
@@ -0,0 +1,91 @@
+/*
+	Copyright 2020 Dynatrace LLC
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+import { SummaryValue } from "@dynatrace/metric-utils";
+import { DataPoint, Histogram } from "@opentelemetry/sdk-metrics-base";
+
+export function estimateHistogram(point: DataPoint<Histogram>): SummaryValue | null {
+	if (point.value.count === 0) {
```

Comment:
Should we export `min=0,max=0,sum=0,count=0` here? A histogram with 0 counts is valid, and not exporting it is dropping data, right?

Reply:
sgtm